src/zm_cluster_info.cpp

```diff
@@ -52,6 +52,7 @@ zmClusterInfo::zmClusterInfo(QWidget *parent) :
     attrHeaders.append("type");
     attrHeaders.append("access");
     attrHeaders.append("value");
+    attrHeaders.append("mfc");
```

Comment:
Can we go with a whole word, like "vendor" or "manufacturer"?

Reply:
Sure, I can change it. However, have you noted that this row is not exposed (while hidden)? I can also make it visible, your call 🙂

Reply:
Oh sorry my bad, I thought it was visible in the UI :D so yeah no need to change it if not visible.